src/Discovery/DiscoveryActionBar.tsx

```diff
@@ -27,8 +29,97 @@ interface Props {
   filtersVisible: boolean;
   setFiltersVisible: (boolean) => void;
   user: User,
+  downloadInProgress: boolean,
+  setDownloadInProgress: (boolean) => void,
+  downloadStatusMessage: {
+    title: string,
+    message: string,
+    active: boolean,
+    url: string
+  },
+  setDownloadStatusMessage: (Object) => void;
 }
 
+const handleDownloadZipClick = async (
+  selectedResources: any[],
+  setDownloadInProgress: (boolean) => void,
+  setDownloadStatusMessage: (object) => void,
+) => {
+  const studyIDs = selectedResources.map((study) => study.project_number);
```

Comment:
I noticed that there is no more checks about manifest field exist for `selectedResources`, is BE job handling it?

Reply:
Yes, I made some changes to return a more user-friendly error if the job fails, and display that: https://github.com/uc-cdis/sower-jobs/blob/ba70c21736323882735c2663639bbdf19964b3f6/export_job/export_zip_from_manifest_job.py#L67